docs/TuxActionStores.md

```diff
@@ -0,0 +1,107 @@
+#Tux ActionStores
+##Table of Contents
+<ol>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#Implementation">Implementation</a></li>
+    <ol style="list-style-type:upper-alpha">
+      <li><a href="#Requiring-ActionStores">Requiring ActionStores</a></li>
+      <li><a href="#Creating-Action-Stores">Creating ActionStores</a>
+      </li>
+      <li><a href="#TuxActionStores-Complete-Example">Tux ActionStores Complete Example</a></li>
+    </ol>
+</ol>
+## <a id="Premise"></a>Premise [#](#Premise)
+>Tux `ActionStores` is a module providing the interface for creating stores that are pre-packaged with a 'register' convenience method for automatically registering the store with actions.
+
+## <a id="Implementation"></a>Implementation [#](#Implementation)
+### <a id="Requiring-ActionStores"></a>Requiring Action Stores [#](#Requiring-Action-Stores)
+
+The interface for creating `ActionStores` is exposed via a constructor function:
+
+```
+  var createActionStore = require('tux/Stores/ActionStores');
+```
+
+### <a id="Creating-Action-Stores"></a>Creating ActionStores [#](#Creating-Action-Stores)
+
+Tux `ActionStores` allow users to register their stores with actions by passing in an object of key-value action properties that will be added to the store being created. If the actions passed in contain a 'register' method, then the `ActionStores` will automatically register all the actions with the new store.
```

Comment:
Change to: `ActionStores` are an implementation on top of the basic `TuxStore`.  Thus, they have all of the same properties.  The only difference is that ActionStores expect to have a `register` method passed in.  If this `register` key is defined than the  ActionStore will invoke the method with the store as the context and use the object the method returns to register the store with `TuxActions`. Let's take a look at an example. (After this will the ActionStore example from below).